datastores/gossip_kv/server/model.rs

```diff
@@ -0,0 +1,109 @@
+use hydroflow::lattices::{DomPair, Max, WithBot};
+use hydroflow::lattices::map_union::MapUnionHashMap;
+use hydroflow::lattices::set_union::SetUnionHashSet;
+
+
+/// Primary key for entries in a table.
+pub type RowKey = String;
+
+/// Value stored in a table. Modelled as a timestamped set of strings.
+///
+/// Each value is timestamped with the time at which it was last updated. Concurrent updates at
+/// the same timestamp are stored as a set.
+pub type RowValue<C> = DomPair<C, SetUnionHashSet<String>>;
+
+/// A map from row keys to values in a table.
+pub type Table<C> = MapUnionHashMap<RowKey, RowValue<C>>;
+
+/// Name of a table in the data store.
+pub type TableName = String;
+
+/// A map from table names to tables.
+pub type TableMap<C> = MapUnionHashMap<TableName, Table<C>>;
+
+/// Timestamps used in the model.
+/// TODO: This will be updated to use a more sophisticated clock type with https://github.com/hydro-project/hydroflow/issues/1207.
```

Comment:
Can either make this a non-doc comment `//` or make it a link `[]`

Reply:
Making it a non-doc.